draft-ietf-quic-transport.md

```diff
@@ -720,9 +720,7 @@ require multiple round trips or retransmissions of this data.
 
 The payload of an Initial packet includes a CRYPTO frame (or frames) containing
```

Comment:
I think "containing a cryptographic handshake message" section makes this sentence more confusing.  Can you remove it?